truss/test_data/truss_container_fs/app/common/serialization.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+import decimal
+import json
+import uuid
+
+import msgpack
+import msgpack_numpy as mp_np
+import numpy as np
+
+
+# mostly cribbed from django.core.serializer.DjangoJSONEncoder
+def truss_msgpack_encoder(obj, chain=None):
```

Comment:
This makes me a bit nervous just copying over the serialization code that is common amongst client and server...

For my understanding, this copy of the serialization code would live on the truss side and do the serializing on the client, while the baseten copy would do the pynode deserialization?

Reply:
Couple things. 
1. The code here is part of test data and is only used for unit tests
2. For a real truss, this is example of code that gets copied into the docker image that's created out of the truss. The deserialization and serialization code here is used for deserializing the binary request to model server and for serializing the response. That part hasn't changed in this PR, but I'm happy to discuss that more.